hnn_core/extracellular.py

```diff
@@ -178,21 +178,19 @@ class ExtracellularArray:
         distance limit between the electrode contacts and the active neuronal
         membrane elements that act as sources of current. The default value of
         0.5 um corresponds to 1 um diameter dendrites.
-    times : None | list of float
+    times : array-like, shape (n_times,) | None
         Optionally, provide precomputed voltage sampling times for electrodes
         at `positions`.
-    voltages : None | list of list of list of float (3D)
+    voltages : array-like, shape (n_trials, n_electrodes, n_times) | None
         Optionally, provide precomputed voltages for electrodes at
-        ``positions``. Note that the size of `voltages` must be:
-        ``(n_trials, n_electrodes, n_times)``, i.e., three-dimensional.
+        ``positions``.
 
     Attributes
     ----------
-    times : list of float
+    times : array, shape (n_times,)
```

Comment:
```suggestion
    times : array-like, shape (n_times,)
```
Consistent with other docstrings?

Reply:
this is actually `array`, not `array-like`. So the difference between the two is that a list that can be cast into an array with `np.array` it is `array-like`. But if we know it's not going to be a list, then it's `array`. Not sure if this is too subtle but other packages use this terminology too ... see MNE, numpy etc.